app/Http/Controllers/Bible/StreamController.php

```diff
@@ -51,29 +51,10 @@ public function index($id = null, $file_id_location = null)
                     ->setStatusCode(HttpResponse::HTTP_NOT_FOUND)
                     ->replyWithError(trans('api.bible_file_errors_404', ['id' => $file_id_location]));
             }
-            $asset_id = $fileset->asset_id;
+
             $current_file = '#EXTM3U';
             foreach ($file->streamBandwidth as $bandwidth) {
-                $current_file .= "\n#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=$bandwidth->bandwidth";
-
-                $transportStream = sizeof($bandwidth->transportStreamBytes)
-                    ? $bandwidth->transportStreamBytes
-                    : $bandwidth->transportStreamTS;
-
-                $extra_args = '';
-                if (sizeof($transportStream) &&
-                    isset($transportStream[0]->timestamp) &&
-                    $transportStream[0]->timestamp->verse_start === 0
-                ) {
-                    $extra_args = '&v0=0';
-                }
-                if ($bandwidth->resolution_width) {
-                    $current_file .= ',RESOLUTION=' . $bandwidth->resolution_width . "x$bandwidth->resolution_height";
-                }
-                if ($bandwidth->codec) {
```

Comment:
would't it be better to simply correct the codec values in the database, rather than munge them in the server?  eg
UPDATE bible_file_stream_bandwidths SET codec='mp4a.40.2' WHERE codec='avc1.4d001f,mp4a.40.2';
UPDATE bible_file_video_resolutions SET codec='avc1.4d001f' WHERE codec='avc1.4d001f,mp4a.40.2';

i don't know that the video one is correct, i'm just guessing...

Reply:
and then we'd need to revise the etl tools which will set the values for future-loaded content

Reply:
@jrstear @bradflood yes, it would be better if we can correct the codec values in the database. But, we could use `resolution_width` column of the `bible_file_stream_bandwidths` entity to determinate if the record is a video or audio. E.g.

```sql
-- audio record
 UPDATE bible_file_stream_bandwidths SET codec='mp4a.40.2' WHERE resolution_width IS NULL;

-- video record
 UPDATE bible_file_stream_bandwidths SET codec='avc1.4d001f,mp4a.40.2' WHERE resolution_width IS NOT NULL;
```

To clarify, video needs both codecs:  `avc1.4d001f,mp4a.40.2` and audio just needs the codec: `mp4a.40.2`

Reply:
@vichugofsl if I update the database as you indicate, are any of these code changes required immediately? I understand that future-loaded content would need the same fix until we can change ETL.

Reply:
@bradflood no, if you update the database, these changes are not required immediately because the codec value would be fetched from database.

Reply:
I executed this query on NEWDATA
UPDATE bible_file_stream_bandwidths SET codec='mp4a.40.2' WHERE resolution_width is null limit 97209;
all audio should have only the mp4 codec.
I will close this PR without approving